blueprints/lib/files/lib/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
why do we have `.jshintrc` there in the first place? :| I was trying to mimic what we had already going on with `JSHint`. Removing `.jshintrc` file is likely necessary but would require changing `blueprints/lib/index.js`. 

I can just remove `.eslintrc.js` and keep `.jshintrc` there and leave the change for another PR.